code/controllers/subsystem/ai_move_test.dm

```diff
@@ -0,0 +1,48 @@
+//Potential implementation of a tile by tile movement via subsystem
+
+SUBSYSTEM_DEF(ai_movement)
+	name = "ai tile by tile"
+	priority = FIRE_PRIORITY_DEFAULT
+	flags = SS_KEEP_TIMING
+	init_order = INIT_ORDER_AI_MOVE
+	wait = 0.5 //This somehow works a lot better than wait = 1, go figure
```

Comment:
this is awful, why the heck are you basically implementing a poorer version of the timer subsystem

Reply:
Removed subsystem and implemented AddTimer instead